proto/catalog.proto

```diff
@@ -37,6 +43,7 @@ message StreamSourceInfo {
   string upsert_avro_primary_key = 7;
   plan_common.FormatType format = 8;
   plan_common.EncodeType row_encode = 9;
+  optional SchemaRegistryNameStrategy name_strategy = 10;
```

Comment:
I *guess* `optional` is unnecessary. The default value (`UNSPECIFIED`) is enough. :eyes:

Reply:
yes, will fix.